routers/repo/http.go

```diff
@@ -314,6 +316,11 @@ func httpBase(ctx *context.Context) (h *serviceHandler) {
 			return
 		}
 
+		if isWiki { // you cannot send wiki operation before create the repository
+			ctx.HandleText(http.StatusForbidden, "Repository not found")
```

Comment:
It should be not found:
```suggestion
			ctx.HandleText(http.StatusNotFound, "Repository not found")
```